Source/carthage/Build.swift

```diff
@@ -38,9 +39,9 @@ public struct BuildCommand: CommandType {
 				return schemeSignals
 					.concat(identity)
 					.on(started: {
-						carthage.println("*** xcodebuild output can be found in \(temporaryURL.path!)")
+						carthage.println(bullets + "xcodebuild output can be found in " + Color.Wrap(foreground: .Yellow).wrap(temporaryURL.path!))
```

Comment:
Only suggestion would be to abstract the specific colours such that we are describing e.g. _the colour for file paths_ or _the colour for errors_ rather than `.Yellow` or `.Green`.